tests/unit/SummaryBusinessHomeJurisdiction.spec.ts

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable max-len */
+import Vue from 'vue'
+import { wrapperFactory } from '../vitest-wrapper-factory'
+import SummaryBusinessHomeJurisdiction from '@/components/ContinuationIn/SummaryBusinessHomeJurisdiction.vue'
+
+describe('Summary Business Home Jurisdiction component', () => {
+  it('renders the component correctly - first summary', async () => {
+    const wrapper = wrapperFactory(
+      SummaryBusinessHomeJurisdiction,
+      null,
+      null,
+      null,
+      null,
+      {
+        isLookup: () => true
```

Comment:
You're basically saying the getter's value is true here right? I like this a lot. I've always went ahead to set the specific store values to make a getter true or false (if that getter relies on store values for example). This is so much easier.

Reply:
Yes.

In this case, it's a local computed/getter and I just wanted to override it without setting any other data.

You can also use this to override store getters in your component.

Reply:
Awesome, sounds good.